emloop/api.py

```diff
@@ -57,17 +64,19 @@ def create_output_dir(config: dict, output_root: str, default_model_name: str='U
         except FileExistsError:
             suffix += 1
 
+    create_config_log(config, output_dir)
     logging.info(f'Created output directory with name {output_dir}')
     return output_dir
 
 
-def config_log_to_output_dir(config: dict, output_dir: str):
-    logging.info('\tOutput dir is: %s', output_dir)
-
-    if not os.path.exists(output_dir):
-        logging.info('\tOutput dir folder "%s" does not exist and will be created', output_dir)
-        os.makedirs(output_dir)
+def create_config_log(config: dict, output_dir: str):
```

Comment:
I still dislike this function (and its name). It does two quite different things none of which is what I would expect from the name (creating a log for logging the configuration?).

lets just rewrite the `create_output_dir` as
```
if output_dir is not None:
    # log that the dir was specified
    # join the output_root and outptu_dir to get output_path
else:
    # log that the dir name will be generated
    # generate the name and join it to get the output_path
# proceed with the dir creation, config dumping, logger set up etc.
```
and this fn is suddenly not needed